### PR TITLE
Add calendar view [AK-63]

### PR DESCRIPTION
**Pull request by szarbartosz**
Calendar lib docs: https://fullcalendar.io/docs

### CODE REVIEW

**Comment by szarbartosz** [comment]
During review You may come across some strange setup - I tried to condense almost all of it in the first commit. The problems is that `Next` does not support the behaviour in which package inside node_modules tries to import css files (for various different reasons - but `ssr` is one of them) - and this calendar library does that. Thankfully I have found this article: https://spacebro.io/articles/fullcalendar-nextjs-10-higher, which fixed the problem.

**Comment by szarbartosz** [comment]
Calendar lib docs: https://fullcalendar.io/docs

File: apps/client/pages/settings.tsx [hunk 1]
```diff
@@ -84,7 +84,7 @@ const Settings: React.FC = (props) => {
             type="submit"
             buttonText="Cancel"
             onClick={() => {
-              router.push('/');
+              router.back();
```
Comment (dimamik) [inline apps/client/pages/settings.tsx:87]: Good point!

File: libs/ui-components/src/lib/TodoList/TaskComponents/TaskModal.tsx [hunk 2]
```diff
@@ -54,13 +54,13 @@ export const TaskModal: React.FC<TaskModalProps> = ({
 
   const [removeTaskMutation] = useRemoveTaskMutation({
     variables: {
-      id: task.id,
+      id: task?.id,
```
Comment (dimamik) [inline libs/ui-components/src/lib/TodoList/TaskComponents/TaskModal.tsx:57]: We probably just shouldn't call this when we don't have a task

File: libs/ui-components/src/lib/TodoList/TaskComponents/TaskModal.tsx [hunk 3]
```diff
@@ -124,45 +124,43 @@ export const TaskModal: React.FC<TaskModalProps> = ({
                   <Thead>
                     <Tr>
                       <Th>Category:</Th>
-                      <Th>{task.category.name}</Th>
+                      <Th>{task?.category?.name}</Th>
```
Comment (dimamik) [inline libs/ui-components/src/lib/TodoList/TaskComponents/TaskModal.tsx:127]: Instead, we can just do smth like:
if (!task) return null;

That's the convention of handling those scenarios.

**Review by dimamik — APPROVED** [review]
Great job!

**Review by karpinsk — APPROVED** [review]
HUGE! 😮 
Thanks a lot, Bartosz!
Later we can make some cleanup, but that`s after Monday and not that important :)

**Pull request merged**